.github/workflows/tests.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         run: sudo xcode-select -s /Applications/Xcode_11.6.app
       - name: Generate Xcode projects and compare against existing record
         run: |
+          rm -rf tests/macos/xcodeproj/*.xcodeproj
```

Comment:
this should go after the `set -euo pipefail`

Reply:
will do! thanks!

Reply:
Should this proj get removed by bazelisk clean ?

Reply:
clean only clears the intermediate products under bazel-out/bazel-bin, certainly not the output right?

Reply:
Oh yeah, you're right. Thanks for the clarification.